product_docs/docs/eprs/7/eprs_rel_notes/eprs_rel_notes_7.9.0.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: "Replication Server 7.9.0 release notes"
+navTitle: Version 7.9.0
+---
+
+Released: 22 Aug 2024
+
+New features, enhancements, bug fixes, and other changes in Replication Server 7.9.0 include the following:
+
+| Type         | Description                                                                                                                                                                                                                                                                                                                | Ticket               |
+|--------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------|
+| Enhancement  | EDB Postgres Replication Server now supports batch replication for Oracle CLOB and BLOB types in Synchronize mode. This significantly reduces the time required to replicate a large number of row changes containing CLOB/BLOB type columns.                                                                              | #102185 <br/> #35513 |
+| Enhancement  | The transaction commit timestamp from the origin node is now preserved when replicating to a target node. This is required to process the timestamp-based UPDATE conflicts when EDB Postgres Replication Server is used in a hybrid cluster where the EPRS Publication node is shared with EDB Postgres Distributed (PGD). |                      |
+| Bug&nbsp;fix | Fixed a corner case issue where the synchronization from the target MMR node failed to resume.                                                                                                                                                                                                                             | #102604 <br/> #35539 |
+| Bug&nbsp;fix | Fixed an issue where some of the processed entries in the `rrep_tx_monitor` table were not cleaned up.                                                                                                                                                                                                                     |                      |
+
+## End-of-support notice
```

Comment:
@mlewandowski-edb @anwar-sahib @zahid75 

This section was still in the Google docs, just double-checking that this still needs to be here, given that the end of standard support lies a year back?

Reply:
Unfortunately, we still do have some customers using Replication Server 6.2. Therefore, I think it is a good idea to keep this section here. I will follow up with some recommended updates to the text here to make it more explicit that they need to upgrade to ensure support.